ontology/yaml/resources/ELECTRICAL/entity_types/UPS.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+UPS_SS:
+  description: "Standard UPS unit."
+  is_canonical: true
+  implements:
+  - UPS
+  - SS  
```

Comment:
would either of the optional fields on SS be required for a UPS?  If so, use them here.

Reply:
explain what you mean